test/util/velero/velero_utils.go

```diff
@@ -98,19 +98,19 @@ var pluginsMatrix = map[string]map[string][]string{
 		"csi":     {"velero/velero-plugin-for-csi:v0.6.0"},
 	},
 	"v1.13": {
-		"aws":       {"velero/velero-plugin-for-aws:v1.9.0"},
-		"azure":     {"velero/velero-plugin-for-microsoft-azure:v1.9.0"},
+		"aws":       {"velero/velero-plugin-for-aws:v1.9.2"},
```

Comment:
Ideally the changes in this file should belong to another PR...

Reply:
Moved to a new commit.